BankexWallet/InterfaceElements/DesignElements/SendTokenViewController/SendTokenViewController.swift

```diff
@@ -127,6 +134,10 @@ Retriable {
     
     var sendingProcess: SendingResultInformation?
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if let favVC = segue.destination as? FavoritesHelperViewController {
+            favVC.allFavorites = self.favorites
```

Comment:
Why should you keep all the array in current controller, if you use it only in another one?

Reply:
This is not the only usage of that array in the current controller, another one is in textfield's delegate method